osu.Game/Rulesets/Objects/Types/IHasComboInformation.cs

```diff
@@ -26,6 +26,14 @@ public interface IHasComboInformation : IHasCombo
         /// </summary>
         int ComboIndex { get; set; }
 
+        Bindable<int> ComboIndexWithOffsetsBindable { get; }
+
+        /// <summary>
+        /// The offset of this combo in relation to the beatmap, with all aggregate <see cref="IHasCombo.ComboOffset"/>s applied.
```

Comment:
So is it *just* the offset, or the combo index *with* the offset applied? Because I'm actively confused by this xmldoc. The first part of this sentence contradicts the other (and the name).

Reply:
Well, the document assigned to both `ComboIndex` and `ComboIndexWithOffsets` need to be rewritten to avoid the term of `offset` in doc.

I'll give it another attempt.

Reply:
Replacing the term "offset" with "index" actually makes things read better:
```diff
diff --git a/osu.Game/Rulesets/Objects/Types/IHasComboInformation.cs b/osu.Game/Rulesets/Objects/Types/IHasComboInformation.cs
index eddf764ae7..29a56fc625 100644
--- a/osu.Game/Rulesets/Objects/Types/IHasComboInformation.cs
+++ b/osu.Game/Rulesets/Objects/Types/IHasComboInformation.cs
@@ -15,21 +15,21 @@ public interface IHasComboInformation : IHasCombo
         Bindable<int> IndexInCurrentComboBindable { get; }

         /// <summary>
-        /// The offset of this hitobject in the current combo.
+        /// The index of this hitobject in the current combo.
         /// </summary>
         int IndexInCurrentCombo { get; set; }

         Bindable<int> ComboIndexBindable { get; }

         /// <summary>
-        /// The offset of this combo in relation to the beatmap.
+        /// The index of this combo in relation to the beatmap.
         /// </summary>
         int ComboIndex { get; set; }

         Bindable<int> ComboIndexWithOffsetsBindable { get; }

         /// <summary>
-        /// The offset of this combo in relation to the beatmap, with all aggregate <see cref="IHasCombo.ComboOffset"/>s applied.
+        /// The index of this combo in relation to the beatmap, with all aggregate <see cref="IHasCombo.ComboOffset"/>s applied.
         /// This should be used instead of <see cref="ComboIndex"/> only when retrieving combo colours from the beatmap's skin.
         /// </summary>
         int ComboIndexWithOffsets { get; set; }
```

@peppy what do you think?

Reply:
Reads better.